LDAR_Sim/src/satellite_company.py

```diff
@@ -52,21 +52,20 @@ def __init__(self, state, parameters, config, timeseries, module_name):
             self.parameters['timesteps'])
         self.timeseries['satellite_flags_redund2'] = np.zeros(
             self.parameters['timesteps'])
-        self.timeseries['satellite_flags_redund3'] = np.zeros(
+        self.timeseries['satellite_flag_wo_vent'] = np.zeros(
             self.parameters['timesteps'])
         self.timeseries['satellite_sites_visited'] = np.zeros(
             self.parameters['timesteps'])
 
         # Assign the correct follow-up threshold
-        if self.config['follow_up_thresh'][1] == "absolute":
-            self.config['follow_up_thresh'] = self.config['follow_up_thresh'][0]
-        elif self.config['follow_up_thresh'][1] == "proportion":
-            self.config['follow_up_thresh'] = get_prop_rate(
-                self.config['follow_up_thresh'][0],
+        if self.config['follow_up']['threshold_type'] == "absolute":
+            self.config['follow_up']['thresh'] = self.config['follow_up']['threshold']
+        elif self.config['follow_up']['threshold_type'] == "proportion":
+            self.config['follow_up']['thresh'] = get_prop_rate(
+                self.config['follow_up']['proportion'],

```

Comment:
Doesn't this need to be done for IFU too? Or is that done elsewhere?

Reply:
Old method obsolete